pkg/kclient/supervisord.go

```diff
@@ -0,0 +1,71 @@
+package kclient
```

Comment:
A lot of the things in this file aren't specific to Kube (other than `AddBootstrapSupervisordInitContainer`), could we move this into somewhere common so that it could be used for Docker and Kube odo?

Reply:
true, i will see where i can move them